packages/datadog-plugin-rhea/src/index.js

```diff
@@ -1,73 +1,92 @@
+'use strict'
+
+let CircularBuffer
```

Comment:
These should not be global as multiple modules with the same name could exist in the same app.